src/tic_tac_clojure/gameplay.clj

```diff
@@ -1,67 +1,28 @@
 (ns tic-tac-clojure.gameplay
   (:gen-class)
-  (:require [tic-tac-clojure.board :refer :all])
-  (:require [tic-tac-clojure.board-render :refer :all])
-  (:require [tic-tac-clojure.game-logic :refer :all])
-  (:require [tic-tac-clojure.message-render :refer :all]))
+  (:require [tic-tac-clojure.board :refer :all]
+            [tic-tac-clojure.cli :refer :all]
+            [tic-tac-clojure.message-render :refer :all]
+            [tic-tac-clojure.player :refer :all]
+            [tic-tac-clojure.round :refer :all]))
 
-(defn convert-to-num
-  [str]
-  (Integer/parseInt str))
-    
-(defn is-number?
-  [str]
-  (do
-    (try (Integer/parseInt str) 
-         true
-         (catch Exception e false))))
-    
-(defn in-range?
-  [start end val]
-  (and (>= val start) (<= val end)))
-
-(defn valid-move?
-  [board num]
-  (and  (in-range? 0 8 num)
-        (empty-space? board num)))
-
-(defn valid-selection?
-  [board str]
-  (and (is-number? str) 
-       (valid-move? board
-                    (convert-to-num str))))
+(def standard-board-length 9)
+            
+(defn- begin-selected-game
+  [num]
+  (case num 
+        1 (play-round (generate-empty-board standard-board-length) 
```

Comment:
From reading the code, I don't know what these numbers mean. You can eliminate these 'magic numbers' by setting constants

Reply:
I've removed the need for a case statement by placing the players' markers and type into hash maps. `player-1` and `player-2` are evaluated and bound locally, so the function doesn't have to do different things based on the number passed in.